extensions/example-extension/renderer.tsx

```diff
@@ -1,23 +1,23 @@
-import { LensRendererExtension, Registry } from "@k8slens/extensions";
-import { ExamplePage, ExampleIcon } from "./page"
+import { LensRendererExtension } from "@k8slens/extensions";
+import { ExampleIcon, ExamplePage } from "./page"
 import React from "react"
 
 export default class ExampleExtension extends LensRendererExtension {
   onActivate() {
     console.log('EXAMPLE EXTENSION RENDERER: ACTIVATED', this.getMeta());
   }
 
-  registerClusterPage(registry: Registry.ClusterPageRegistry) {
-    this.disposers.push(
-      registry.add({
+  registerClusterPage() {
```

Comment:
Maybe just `clusterPages()`.

Reply:
I was thinking `clusterPages()` too, but I can see it might be a bit vague. We want it to be clear to the developer that this is a method that is called (only once, on initialization) by the extensions manager. `initClusterPages()`? `setClusterPages()`? I think `registerClusterPages()` works too. (it does need to be plural though, right?)